Doc/source/designspaceLib/index.rst

```diff
@@ -120,6 +120,34 @@ any of the UFOs. If the lib key is empty or not present in the Designspace, all
 glyphs should be exported, regardless of what the same lib key in any of the
 UFOs says.
 
+public.fontInfo
+-----------------------
+
+This lib key, when included in the ``<lib>`` element inside an ``<instance>`` or 
+``<variable-font>`` tag, allows for direct manipulation of font info data in 
+instances and variable fonts. The lib value should follow the 
```

Comment:
'should' should be 'must'

Reply:
anything that is a hard requirement should use "must". Should implies that a thing can be optional. @colinmford see https://unifiedfontobject.org/versions/ufo3/#terminology and the linked RFC 2119 there.